src/lib/Microsoft.Fx.Portability/Reporting/IReportWriter.cs

```diff
@@ -3,13 +3,14 @@
 
 using Microsoft.Fx.Portability.ObjectModel;
 using System.IO;
+using System.Threading.Tasks;
 
 namespace Microsoft.Fx.Portability.Reporting
 {
     public interface IReportWriter
     {
         ResultFormatInformation Format { get; }
 
-        void WriteStream(Stream stream, AnalyzeResponse response);
+        Task WriteStreamAsync(Stream stream, AnalyzeResponse response);
```

Comment:
Oh, I see. Some of those other async changes were to support this interface change.